ci/azure-pipeline-connectors-mondodb.yaml

```diff
@@ -0,0 +1,18 @@
+trigger:
```

Comment:
File name should be `ci/azure-pipeline-connectors-mongodb.yaml`

Reply:
Nice catch! Fixed it.